stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py

```diff
@@ -21,7 +21,6 @@ class ProductPrintingQty(models.TransientModel):
     uom_id = fields.Many2one(
         "uom.uom",
         string="Unit of Measure",
-        related="move_line_id.product_uom_id",
```

Comment:
Is this change contemplated for the other wizard calls?

Reply:
Yes, it is filled from move_line_ids in this method: https://github.com/OCA/stock-logistics-barcode/blob/1b8a23f172608a4afc2e803e7e312a1901c96ca8/stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py#L135

IDK why it was related if it was filled there :sweat_smile: